### DISCUSSION TITLE
Meeting agenda w/c 27th June

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#11] [Meeting Agenda]
Let's start pulling together an agenda for our next meeting. I'd like to discuss:

- A very high-level project plan for refining the Contribution process (I don't have a proposed plan yet, so feel free to create one and share in this discussion if you like)
- I'd like to start brain-storming around the Contribution process, by determining the questions we should be asking, see #10

**Comment by eminty69** [comment]
Are there any projects currently going through this process?   It might be useful to see what they are and where they are in process

**Reply by ColinEberhardt** [reply]
Yes, there is a pipeline, which is visible on GitHub https://github.com/finos/community/issues?q=is%3Aissue+label%3Acontribution+

**Comment by jonfreedman** [comment]
On the topic of a high level plan, I'd start with the contributor experience: what do we want this to look like? I'd propose a single 'thing' - the fact we started with two separate links to two different documents being a counterexample
Then figure out what if anything needs to be private during the process (@mindthegab some legal input maybe required)

**Reply by ColinEberhardt** [reply]
Good idea focussing on the contributor experience.

> the fact we started with two separate links to two different documents being a counterexample

Can you please elaborate? Where are these links?

**Reply by jonfreedman** [reply]
I'm referring to [contribution process](https://community.finos.org/docs/governance/Software-Projects/contribution) and [incubation checklist](https://community.finos.org/docs/governance/Software-Projects/stages/incubating)
There is "Optionally FINOS Members can carry out a pre-incubation so called [Formation stage](https://community.finos.org/docs/governance/software-projects/project-lifecycle/#forming-projects-optional)" but if you assume that doesn't apply then in theory the contributor has to follow both docs in parallel

**Comment by ColinEberhardt** [comment]
This meeting has now taken place, minutes can be found here: https://github.com/finos/technical-steering-committee/pull/14
